tooling/template/app/[[...permalink]]/page.tsx

```diff
@@ -12,11 +12,13 @@ import {
   RenderEngine,
 } from "@opengovsg/isomer-components"
 
+export const dynamic = "force-static"
```

Comment:
do we need this statement across the files? could you summarise what this implies?

Reply:
I would think that having the following in the next.config will auto-force all to be static

```
const nextConfig = {
  output: "export",
```

Reply:
@harishv7 yeah i would think so too but somehow not including this will cause an issue during `next build`